app/go_to_date/static/popup.js

```diff
@@ -0,0 +1,7 @@
+function openForm() {
+    document.getElementById("myForm").style.display = "block";
```

Comment:
Use CSS and create class which the property `{ display: block }`, then use `toggleClass` to add/remove this class from `openForm`.

Don't edit the style directly from the JavaScript, as we try not to mix logic with styling.

Reply:
I got a bit confused,see toggleClass belongs to jquery library which uses a different click function (not addEventListener), so should I use something like that?:
$(document).ready(function(){
    $("button").click(function(){
      $("p").toggleClass("main");

Reply:
Yeah sure. I meant you should use `element.classList.toggle`